fvtest/vmtest/CMakeLists.txt

```diff
@@ -30,16 +30,16 @@ target_link_libraries(omrvmtest
 	omrGtestGlue
 	omrtestutil
 	omrcore
-	omrtrace
-	omrgc
-	j9hookstatic
-	j9thrstatic
 	omrutil
-	omrport
 	j9hashtable
 	j9avl
 	j9pool
-
+	omrvmstartup
+	${OMR_HOOK_LIB}
+	${OMR_GC_LIB}
+	${OMR_PORT_LIB}
+	${OMR_TRHREAD_LIB}
```

Comment:
typo

Reply:
well i guess they dont need the thread lib then 😄

Reply:
This may be the cause of the AIX compile failure :)